libfreerdp/core/connection.c

```diff
@@ -773,7 +773,7 @@ BOOL rdp_client_connect_auto_detect(rdpRdp* rdp, wStream* s)
 		{
 			if (channelId == rdp->mcs->messageChannelId)
 			{
-				UINT16 securityFlags;
+				UINT16 securityFlags = 0;
```

Comment:
@bmiklautz Yes, there was a warning during android compile. I've just initialized it for every case where the read / write functions are used.